applications/tari_indexer/web_ui/src/routes/Events/Events.tsx

```
@@ -144,74 +156,104 @@ function EventsLayout() {
+                <TableCell>Transaction</TableCell>
+                <TableCell>Substate Id</TableCell>
+                <TableCell>Template</TableCell>
+                <TableCell>Payload</TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
               {events.map((row: any) => (
                 <TableRow
-                  sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+                  sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
                 >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)